Plugins/Flow.Launcher.Plugin.Sys/Languages/ko.xaml

```diff
@@ -15,6 +18,17 @@
     <system:String x:Key="flowlauncher_plugin_sys_setting">이 프로그램을 조정합니다</system:String>
     <system:String x:Key="flowlauncher_plugin_sys_sleep">PC를 절전모드로 전환</system:String>
     <system:String x:Key="flowlauncher_plugin_sys_emptyrecyclebin">휴지통 비우기</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_openrecyclebin">휴지통 열기</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_openmycomputer">내 PC</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_openmydesktop">바탕 화면</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_libraries">라이브러리</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_explorer">파일 탐색기</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_pictures">사진</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_music">음악</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_videos">비디오</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_downloads">다운로드</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_documents">문서</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_indexoption">색인 옵션</system:String>

```

Comment:
You may need to copy this back to crowdin if we merge🤔.

Reply:
ok.